test/SlowTests/Voron/Storage/StorageReportGenerationTests.cs

```diff
@@ -439,7 +439,7 @@ public void TreeReportContainsPartialInfoAboutStreams(int numberOfStreams, int s
                 var streamsSizeInMb = new Size(treeReport.Streams.Streams[0].AllocatedSpaceInBytes, SizeUnit.Bytes).GetValue(SizeUnit.Megabytes);
                 var fullStreamsSizeInMb = new Size(fullTreeReport.Streams.AllocatedSpaceInBytes, SizeUnit.Bytes).GetValue(SizeUnit.Megabytes);
                 
-                Assert.Equal(streamsSizeInMb,fullStreamsSizeInMb);
+                Assert.True(streamsSizeInMb == fullStreamsSizeInMb || streamsSizeInMb + 1 == fullStreamsSizeInMb || streamsSizeInMb - 1 == fullStreamsSizeInMb);
```

Comment:
For the test assertion it's okay but think it would be easier to express it as `Math.Abs(fullStreamsSizeInMb - streamsSizeInMb) <= 1`.